synapse/rest/client/versions.py

```diff
@@ -49,24 +49,11 @@ def on_GET(self, request):
                     "r0.3.0",
                     "r0.4.0",
                     "r0.5.0",
+                    "r0.6.0",
                 ],
                 # as per MSC1497:
                 "unstable_features": {
-                    # as per MSC2190, as amended by MSC2264
-                    # to be removed in r0.6.0
-                    "m.id_access_token": True,
-                    # Advertise to clients that they need not include an `id_server`
-                    # parameter during registration or password reset, as Synapse now decides
-                    # itself which identity server to use (or none at all).
-                    #
-                    # This is also used by a client when they wish to bind a 3PID to their
-                    # account, but not bind it to an identity server, the endpoint for which
-                    # also requires `id_server`. If the homeserver is handling 3PID
-                    # verification itself, there is no need to ask the user for `id_server` to
-                    # be supplied.
-                    "m.require_identity_server": False,
-                    # as per MSC2290
-                    "m.separate_add_and_bind": True,
+                    "m.lazy_load_members": True,
```

Comment:
this is something new? can it have a comment with an MSC link or so?

Reply:
Ah, good catch. Looks like this was removed in https://github.com/matrix-org/synapse/commit/dc3f9987061e24bec77ad9e26a7679a1907cee4f#diff-e591d42d30690ffb79f63bb726200891 and was accidentally kept while rebasing this branch.